terraform/modules/alerting/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_sns_topic" "sns_general_availability" {
+  name = "${var.sns_general_availability_topic_name}"
+
+  # https://github.com/hashicorp/terraform/issues/10697#issuecomment-312029088
+  # SNS not supported in all regions
+  # https://docs.aws.amazon.com/sns/latest/dg/sms_supported-countries.html
```

Comment:
Not a blocker: meant to move these two lines to the `description` of the `general_availability_protocol` `variable`.